transport/conn.go

```diff
@@ -158,26 +158,52 @@ func (c *connReader) parse(op frame.OpCode) frame.Response {
 		return ParseError(&c.buf)
 	case frame.OpReady:
 		return ParseReady(&c.buf)
+	case frame.OpResult:
+		return c.parseResult()
 	default:
 		log.Fatalf("not supported %d", op)
 		return nil
 	}
 }
 
+func (c *connReader) parseResult() frame.Response {
+	resultKind := c.buf.ReadInt()
+	switch resultKind {
+	case VoidKind:
+		return ParseVoidResult(&c.buf)
+	case RowsKind:
+		return ParseRowsResult(&c.buf)
+	case SetKeySpaceKind:
+		return ParseSetKeyspaceResult(&c.buf)
+	case PreparedKind:
+		return ParsePreparedResult(&c.buf)
+	case SchemaChangeKind:
+		return ParseSchemaChangeResult(&c.buf)
+	default:
+		return nil
+	}
+}
+
 type Conn struct {
 	conn net.Conn
 	w    connWriter
 	r    connReader
 
+	cfg ConnConfig
+
 	stream StreamIDAllocator
 }
 
+type ConnConfig struct {
+	opts frame.StartupOptions
```

Comment:
What's that?

Reply:
We need proper creation of StartupOptions.
This is beyond the scope, add fixed value where needed.

Reply:
Startup options are neccessary to init connection. I thought if it would be better to pass opts as a parameter to init() and in consequence to WrapConn(...), but finally decided to place it in config, because many connections will probably have the same configs. 
ConnConfig will be filled with other variables, but for now I needed only options.